app/src/main/java/org/jellyfin/androidtv/ui/preference/category/playback.kt

```diff
@@ -53,11 +53,14 @@ fun OptionsScreen.playbackCategory(
 	seekbar {
 		setTitle(R.string.pref_next_up_timeout_title)
 		setContent(R.string.pref_next_up_timeout_summary)
-		min = 3000
+		min = 0
 		max = 30000
-		increment = 1000
+		increment = 3000
 		valueFormatter = object : DurationSeekBarPreference.ValueFormatter() {
-			override fun display(value: Int) = "${value / 1000}s"
+			override fun display(value: Int) = when {
+				value == 0 -> "∞" // disables the timer and displays infinity character
```

Comment:
It might be better to create a constant for the 0/infinite value somewhere and use it both here and in the nextupfragment.

Additionally, the string should be in strings.xml so use getString here for the infinite character.